docs/devel/generating-clientset.md

```diff
@@ -7,67 +7,54 @@ cycle of the generated clientsets.
-  adds additional methods to Pod's client. As a convention, we put the expansion
-  interface and its methods in file ${TYPE}_expansion.go. In most cases, you
-  don't want to remove existing expansion files. So to make life easier,
-  instead of creating a new clientset from scratch, ***you can copy and rename an
-  existing clientset (so that all the expansion files are copied)***, and then run
-  client-gen.
+3. ***Adding expansion methods***: client-gen only generates the common methods,
+   such as CRUD. You can manually add additional methods through the expansion
+   interface. For example, this
+   [file](../../pkg/client/clientset_generated/release_1_5/typed/core/v1/pod_expansion.go)
+   adds additional methods to Pod's client. As a convention, we put the
+   expansion interface and its methods in file ${TYPE}_expansion.go. In most
+   cases, you don't want to remove existing expansion files. So to make life
+   easier, instead of creating a new clientset from scratch, ***you can copy and
+   rename an existing clientset (so that all the expansion files are copied)***,
+   and then run client-gen.
 
-- Generating fake clients for testing purposes: client-gen will generate a fake
-clientset if the command line argument `--fake-clientset` is set. The fake
-clientset provides the default implementation, you only need to fake out the
-methods you care about when writing test cases.
-
-The output of client-gen includes:
+## Output of client-gen
 
 - clientset: the clientset will be generated at
-`pkg/client/clientset_generated/` by default, and you can change the path via
-the `--clientset-path` command line argument.
+  `pkg/client/clientset_generated/` by default, and you can change the path via
+  the `--clientset-path` command line argument.
 
-- Individual typed clients and client for group: They will be generated at `pkg/client/clientset_generated/${clientset_name}/typed/generated/${GROUP}/${VERSION}/`
+- Individual typed clients and client for group: They will be generated at
+  `pkg/client/clientset_generated/${clientset_name}/typed/generated/${GROUP}/${VERSION}/`
 
 ## Released clientsets
 
-At the 1.2 release, we have two released clientsets in the repo:
-internalclientset and release_1_2.
-
-- internalclientset: because most components in our repo still deal with the
-internal objects, the internalclientset talks in internal objects to ease the
-adoption of clientset. We will keep updating it as our API evolves. Eventually
-it will be replaced by a versioned clientset.
-
-- release_1_2: release_1_2 clientset is a versioned clientset, it includes
-clients for the core v1 objects, extensions/v1beta1, autoscaling/v1, and
-batch/v1 objects. We will NOT update it after we cut the 1.2 release. After the
-1.2 release, we will create release_1_3 clientset and keep it updated until we
-cut release 1.3.
-
+To get stable clientsets, please refer to the [client-go
```

Comment:
Probably add a note saying new code should use client-go and we are in the process of updating existing code to use client-go. Perhaps also add link to the issue?